roamance/src/components/auth/AuthLayout.tsx

```
@@ -0,0 +1,36 @@
+import { Footer } from '@/components/footer';
+
+interface AuthLayoutProps {
+  children: React.ReactNode;
+}
+
+export function AuthLayout({ children }: AuthLayoutProps) {
+  return (
+    <div className="min-h-screen flex flex-col relative bg-gradient-to-br from-background via-background to-background/80 overflow-hidden">
+      {/* Decorative elements */}
+      <div className="absolute inset-0 overflow-hidden pointer-events-none">
+        <div className="absolute -top-[30%] -right-[10%] w-[70%] h-[70%] rounded-full bg-primary/5 blur-3xl"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)